### PR TITLE
fix: bundle swiftshader and ANGLE libraries on mac

### LINKED ISSUE #18639: Consistent crash when hardware acceleration is disabled
### Preflight Checklist

* [x] I have read the [Contributing Guidelines](https://github.com/electron/electron/blob/master/CONTRIBUTING.md) for this project.
* [x] I agree to follow the [Code of Conduct](https://github.com/electron/electron/blob/master/CODE_OF_CONDUCT.md) that this project adheres to.
* [x] I have searched the issue tracker for an issue that matches the one I want to file, without success.

### Issue Details

* **Electron Version:**
  * 6.0.0-beta.5 (also present in beta.6)
* **Operating System:**
  * macOS 10.14.5
* **Last Known Working Electron version:**
  * 6.0.0-beta.4

### Expected Behavior
Electron should not crash after calling `app.disableHardwareAcceleration()`

### Actual Behavior
Electron crashes (main process) when started with hardware acceleration disabled.

```
[16974:0604/215819.260822:FATAL:gpu_data_manager_impl_private.cc(901)] The display compositor is frequently crashing. Goodbye.
```

### To Reproduce

Reproducible trivially with electron-quick-start

```sh
$ git clone https://github.com/adill/electron-quick-start -b macOS-no-gpu-crash
$ npm install
$ npm start
```

### PR DESCRIPTION
**Pull request by nornagon**
#### Description of Change
Fixes #18639. Not sure what exactly caused this to start breaking now, but these libraries appeared in https://chromium-review.googlesource.com/c/chromium/src/+/955949/ and got enabled by default on mac here https://chromium-review.googlesource.com/c/chromium/src/+/1005017/. The call during gpu process startup that was causing this crash was added here https://chromium-review.googlesource.com/c/chromium/src/+/1599993.

#### Checklist

- [x] PR description included and stakeholders cc'd
- [ ] `npm test` passes
- [ ] tests are [changed or added](https://github.com/electron/electron/blob/master/docs/development/testing.md)
- [ ] relevant documentation is changed or added
- [x] PR title follows semantic [commit guidelines](https://github.com/electron/electron/blob/master/docs/development/pull-requests.md#commit-message-guidelines)
- [x] [PR release notes](https://github.com/electron/clerk/blob/master/README.md) describe the change in a way relevant to app developers, and are [capitalized, punctuated, and past tense](https://github.com/electron/clerk/blob/master/README.md#examples).

#### Release Notes

Notes: Fixed crash when starting with `--disable-gpu`.

### CODE REVIEW

**Comment by nornagon** [comment]
I think this supersedes https://github.com/electron/electron/pull/18778

File: BUILD.gn [hunk 1]
```diff
@@ -677,8 +722,10 @@ if (is_mac) {
       ":electron_lib",
     ]
     deps = [
+      ":electron_angle_library",
```
Comment (deepak1556) [inline BUILD.gn]: Any reason to isolate these in targets like `electron_angle_library`, `electron_swiftshader_library` ? Can't they be combined into `electron_framework_libraries` ?

Comment (nornagon) [reply BUILD.gn]: I just copy/pasted this from content_shell. Left it the same because that was easiest 🤷‍♂

**Comment by codebytere** [comment]
@nornagon can you please backport this to 6?

**Pull request merged**